pkg/suite/suite.go

```diff
@@ -212,17 +212,32 @@ func (s *suite) Run(t *testing.T, suiteName string) {
 				)
 			},
 			func(wcClient *clusterclient.Client) {
+				skipDefaultAppsApp, err := state.GetCluster().UsesUnifiedClusterApp()
+				Expect(err).NotTo(HaveOccurred())
+
 				logger.Log("Waiting for all default apps to be ready")
-				defaultAppsAppName := fmt.Sprintf("%s-%s", state.GetCluster().Name, "default-apps")
 
-				Eventually(wait.IsAppDeployed(state.GetContext(), state.GetFramework().MC(), defaultAppsAppName, state.GetCluster().Organization.GetNamespace())).
-					WithTimeout(30 * time.Second).
-					WithPolling(50 * time.Millisecond).
-					Should(BeTrue())
+				defaultAppsSelectorLabels := cr.MatchingLabels{
+					"giantswarm.io/cluster":        state.GetCluster().Name,
+					"app.kubernetes.io/managed-by": "Helm",
+				}
+
+				if !skipDefaultAppsApp {
+					defaultAppsAppName := fmt.Sprintf("%s-%s", state.GetCluster().Name, "default-apps")
+
+					Eventually(wait.IsAppDeployed(state.GetContext(), state.GetFramework().MC(), defaultAppsAppName, state.GetCluster().Organization.GetNamespace())).
```

Comment:
Do we have a wait case for when skipDefaultAppsApp is true?

Reply:
No, because we already wait for the cluster to be ready.